.github/workflows/pypi.yaml

```diff
@@ -0,0 +1,30 @@
+name: pypi
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  pypi:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: '1'
+      - name: Extract version
+        shell: bash
+        run: echo "::set-output name=tag::$(echo ${GITHUB_REF##*/} | sed s/^v//g)"
+        id: version
+      - name: Python re-generate client
+        run: make client-python PACKAGE_VERSION=${{ steps.version.outputs.tag }}
+      - name: Python build package
+        run: make package-python 
+      - name: Python publish package
+        if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags')
```

Comment:
How can it be a `push` event when you're only triggered by `published`?

Reply:
true, I was triggering it manually - I'll remove the first part and just keep the tags check